### DISCUSSION TITLE
Increased Supavisor Client Connection Limits Across Paid Plans

### DISCUSSION THREAD

**Discussion by w3b6x9** [supabase/supabase#22457] [Changelog]
Supavisor, Supabase's multi-tenant connection pooler deployed to regional clusters, became production ready back in December 2023. You can read the announcement [here](https://supabase.com/blog/supavisor-postgres-connection-pooler).

Since then, we've migrated Supabase projects from PgBouncer, single tenant connection pooler deployed to the project's instance, to Supavisor.

However, we kept the previous client connection limits from PgBouncer during the transition across all compute instances.

Today, we're happy to announce that we've increased this limit for compute instances `Small`, `Medium`, `Large`, and `XL` so your projects can take advantage of additional client connections while pricing remains unchanged. These new limits have already been applied to all existing projects and any new projects spun up.

Here's a quick breakdown:

| Compute Size | Previous Client Limits | New Client Limits |
| -------------- | ---------------------  | ----------------- |
| Small                | 200                               | 400                       |
| Medium           |  200                               |  600                      |
| Large               |  300                               |  800                      |
| XL                    |  700                               | 1,000                     |

For a more complete breakdown of your compute instance resources head over to the [Compute Add-ons](https://supabase.com/docs/guides/platform/compute-add-ons) page.

**Comment by relferreira** [comment]
Great news!!
I have many problems with max_connections errors. Is there a way to monitor those errors? If not, is there any plan to add logs or ways to debug this on the dashboard?

**Reply by TheOtherBrian1** [reply]
[Preventing "Max client connections reached" Errors Trouble Shooting Guide](https://github.com/orgs/supabase/discussions/22305)
